tests/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtensionTest.php

```diff
@@ -219,6 +222,23 @@ public function testLoadDefaultConfig()
         $this->extension->load(self::DEFAULT_CONFIG, $containerBuilder);
     }
 
+    /**
+     * @group mongodb
+     */
+    public function testLoadDefaultConfigWithOdm()
```

Comment:
```suggestion
    public function testLoadDefaultConfigWithMongoDbOdm()
```
Because there are other ones.

Reply:
@Toflar